Content.Shared/Cognislime/SharedCognislimeSystem.cs

```diff
@@ -0,0 +1,28 @@
+using Content.Shared.DoAfter;
+using Robust.Shared.Serialization;
+
+
+namespace Content.Shared.Cognislime;
+
+/// <summary>
+/// Makes objects sentient.
+/// </summary>
+public abstract class SharedCognislimeSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+    }
+
+    [Serializable, NetSerializable]
```

Comment:
move this past the system declaration so its just in the namespace directly

Reply:
Moving `[Serializable, NetSerializable]` right below the xml summary and above the system declaration throws an error since the doafterevent isn't serializable any more. I'm gonna mark this as resolved, comment again if this still needs to be fixed.